data_validation/combiner.py

```diff
@@ -446,55 +447,63 @@ def _get_summary(
 ):
     """Logs a summary report/stats of row validation results."""
     try:
-        if (
-            not result_df.empty
-            and result_df.loc[0, consts.VALIDATION_TYPE] == consts.ROW_VALIDATION
-        ):
-            # Vectorized calculations for all counts.
-            success_condition = (
-                result_df[consts.VALIDATION_STATUS] == consts.VALIDATION_STATUS_SUCCESS
-            )
-            fail_condition = ~success_condition  # Invert success for fail condition.
+        if not result_df.empty:
```

Comment:
Here I would be tempted to flip the condition and return out. For example:
```
if result_df.empty:
    return
```

That means you won't need to indent the body any further than it already was.

Reply:
Great catch, agreed!